scripts/environment-deploy.sh

```diff
@@ -53,6 +53,56 @@ pushd "$SOLUTION_DIR/post-deployment" > /dev/null
 $EXEC sls invoke -f postDeployment -l -s "$STAGE"
 popd > /dev/null
 
+# Get the first header (not Changelog) in CHANGELOG.md
+versionLine="$(cat CHANGELOG.md | grep -m 1 "[0-9]\.[0-9]\.[0-9]\|Beta")"
```

Comment:
Use `[0-9]+` for the first number in case our versioning crosses `9`.

Reply:
Got it. I am going to add that to all three digits in case we need it.